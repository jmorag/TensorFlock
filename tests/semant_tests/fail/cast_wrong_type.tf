main = cast 3.14;
