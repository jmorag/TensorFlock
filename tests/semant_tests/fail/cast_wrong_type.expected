Fatal error: exception Failure("Expected type Nat but instead received T<>")
